README.md

```diff
@@ -43,12 +43,14 @@ cargo install wrangler
     with the `"type"` declared there.
 
   - ### 🔧 `config`
-    Configure your global Cloudflare user. You will need to pass your email and API key:
+    Configure your global Cloudflare user. This is an interactive command that will prompt you for your email and API key:
 
     ```
-    wrangler config <email> <api_key>
+    wrangler config
```

Comment:
I also recommend putting examples of the prompts in here :)

Reply:
demo gif? might be too much overhead, but it is fun to learn how to make those

Reply:
Here's what I wrote in the docs if you want to re-use:

> To configure Wrangler to deploy to your Cloudflare Workers account, set up your default credentials via the config subcommand. You should only need to do this once. Running wrangler config will prompt you interactively for your email and API key:

```sh
$ wrangler config
Enter email:
foo@bar.com
Enter api key:
123456abcdef
```